devel.json

```diff
@@ -25,6 +25,21 @@
         }
       }
     },
+    "permissions": {
+      "org_integrity": {
+        "orgs": [
+          {
+            "name": "my-org-1"
+          },
+          {
+            "name": "my-org-2",
+            "host": "https://github.my-company.com",
+            "collaborator_types": ["direct", "outside"],
+            "repos": ["repo1", "repo2"]
+          }
+        ]
```

Comment:
This should be changed back to using `url` rather than name and host.

Reply:
To discuss. Same as per https://github.com/ComplianceAsCode/auditree-arboretum/pull/38#discussion_r559445782.

Reply:
Fixed after discussing